dockerCompose/dockerComposeCompletionItemProvider.ts

```diff
@@ -55,12 +59,14 @@ export class DockerComposeCompletionItemProvider implements CompletionItemProvid
         return Promise.resolve([]);
     }
 
-    private suggestKeys(word: string): CompletionItem[] {
-        return Object.keys(DOCKER_COMPOSE_KEY_INFO).map(ruleName => {
+    private suggestKeys(word: string, useV2Schema: boolean = false): CompletionItem[] {
```

Comment:
I actually just sent out a new commit that addressed your feedback, and cleaned up the way that the version keys are composed, that I think makes it a little easier to read. Let me know what you think!